docs/install.rst

```diff
@@ -0,0 +1,352 @@
+.. highlight:: bash
+
+.. _install:
+
+==================
+Quick installation
+==================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Preface
+=======
+
+This part of the documentation covers the installation of CrateDB on Linux,
+macOS and Windows systems.
+The first step to using any software package is getting it properly installed.
+Please read this section carefully.
+
+
+Introduction
+============
+
+By installing CrateDB the way as outlined in this document, you will be able to
+quickly setup and run a single-node cluster.
+
+After the installation is finished, the CrateDB service should be up and
+running. Now, you can access the :ref:`Admin UI <admin-ui:index>` from your
+local machine by navigating to::
+
+    http://localhost:4200/
+
+After successfully installing the software, you might want to follow up with
+:ref:`taking the guided tour <use>`.
+
+Also, you might enjoy to be guided into further information about how to
+quickly adjust the :ref:`configuration settings <configure>`, read more
+details about the :ref:`reference:config` of CrateDB, the background about
+:ref:`howtos:bootstrap-checks`, multi-node configuration within the section
+about :ref:`howtos:clustering` and :ref:`howtos:going-into-production`. When
+operating a CrateDB cluster in production, `performance tuning`_ will also be
+of interest.
+
+.. _performance tuning: https://crate.io/docs/crate/howtos/en/latest/performance/
+
+.. NOTE::
+
+    To add additional CrateDB nodes to this kind of cluster in order to make it
+    form a multi-node cluster, you will need to remove the cluster state after
+    changing the configuration.
```

Comment:
Can you point out some details how to exactly do that?

Reply:
Both https://github.com/crate/crate/issues/10058 and https://github.com/crate/crate-howtos/issues/174 might be related to this topic, right?

Reply:
correct 👍

Reply:
Thank you. So, we might want to bring the quintessence of the content at [1] into a separate document here and link to this from the section referenced above?

[1] https://community.crate.io/t/3-node-cluster-not-forming-after-install-azure-ubuntu-vms/426/2

Reply:
Similar here. I find this note here at this point a bit premature and distracting. I think we should move this further to the bottom - if at all

Reply:
To elaborate a bit on my thoughts here: I think we've to be careful and avoid overloading a user with information that is not yet relevant for them. Mentioning a lot of things upfront gives the impression of things being complicated. If we gradually introduce the content as it becomes relevant the users are less intimidated.

Reply:
Thank you for your clear statement. I had the same feeling while editing this document but didn't know about the background why this paragraph was meant to be placed here. I will be more than happy to improve it and to put all relevant admonitions in this space to the end of the document.